File: PythonCode/1036.py
#Questão: Formula de bhaskara
A, B, C = input().split()
A = float(A)
B = float(B)
C = float(C)
delta = (B**2) - 4 * A * C
if A != 0 and delta > 0:
    delta1 = delta ** (1 / 2)
    R1 = (-B + delta1) / (2 * A)
    R2 = (-B - delta1) / (2 * A)
    print("R1 = %.5f" % R1)
    print("R2 = %.5f" % R2)
else:
    print("Impossivel calcular")

File: C++Code/1036.cpp
//Questão: Formula de bhaskara
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main(){
    double a,b,c,d,delta,delta1;
    cin >> a>>b>>c;
    delta = (pow(b,2)-4*a*c);
    if (a != 0 && delta > 0){
        delta1 = pow(delta,0.5);
        cout << fixed << setprecision(5)<< "R1 = "<< (-b + delta1) / (2 * a) << endl;
        cout << fixed << setprecision(5)<< "R2 = "<< (-b - delta1) / (2 * a) << endl;
    }
    else{
        cout <<"Impossivel calcular"<< endl;
    }
    return 0;
}